### PR TITLE
Added Example to :class:`~.PMobject`

### PR DESCRIPTION
**Pull request by MathInvariance**
## Explanation for Changes

[Documentation Reference](https://manimce--####.org.readthedocs.build)

## Checklist
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [x] I have written a descriptive PR title (see top of PR template for examples)
- [x] I have written a changelog entry for the PR or deem it unnecessary
- [ ] My new functions/classes either have a docstring or are private
- [ ] My new functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [x] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [ ] The PR is labeled correctly
- [ ] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

**Comment by kolibril13** [comment]
Reference here:
https://manimce--1698.org.readthedocs.build/en/1698/reference/manim.mobject.types.point_cloud_mobject.PMobject.html?highlight=pmobject#manim.mobject.types.point_cloud_mobject.PMobject

File: manim/mobject/types/point_cloud_mobject.py [hunk 1]
```diff
@@ -21,6 +21,34 @@
...
+            def construct(self):
+
+                pG = PGroup()  # This is just a collection of PMobject's
+
+                # As the scale factor increases, the number of points
+                # removed increases.
+                for sf in range(1, 9 + 1):
+                    p = PointCloudDot(density=20, radius=1).thin_out(sf)
+                    # PointCloudDot is a type of PMobject
+                    # So, it can be added to a PGroup
```
Comment (kolibril13) [inline manim/mobject/types/point_cloud_mobject.py:42]:
```suggestion
                    # and can there fore be added to a PGroup
```

**Comment by kolibril13** [comment]
Looks good to me in general!
Locally, I just tried to use only `Group` instead of `PGroup`.
The result was the same.
So, can you maybe add something to the PGroup example, that is unique to a PGroup and can not be achieved with a normal group?

**Comment by MathInvariance** [comment]
> So, can you maybe add something to the PGroup example, that is unique to a PGroup and can not be achieved with a normal group?

Since this is an example for PMobject, wouldn't it be sufficient to show that PointCloudDot can access functions that are meant for a PMobject. Additionally, the only reason I have a PGroup in this example is to neatly organize all the PMobjects without manually setting individual positions.

**Review by behackl — APPROVED** [review]
I think this is fine.

We'll need to consider further steps to make `PMobject` more clear anyways.

**Pull request merged**